boxsdk/object/search.py

```diff
@@ -266,6 +267,12 @@ def query(
             The direction to display the sorted search results. Can be set to `DESC` for descending or `ASC` for ascending.
         :type direction:
             `unicode` or None
+        :param include_recent_shared_links:
+            If False, the search results will only include items that are either owned by the user or items that the
+            user has been explicitly collaborated on.
+            If True, the search results may also include items that are only accessible via recently used shared links.
```

Comment:
Should we also mention that the results will be returned with a slightly different format when this param is True?